docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: GX Overview
+---
+
+import PythonVersion from '/docs/components/_python_version.mdx';
+
+Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, GX applies Expectations to data instead of code. For example, you could define an Expectation that a column contains no null values, and GX would run that Expectation against your data, and report if a null value was found.
+
+## Before you start
+
+Before you've even installed GX, working with GX starts with data.  Specifically, data you care about enough for quality to be important.  We call your data, in its original format, your **Source Data**.  The combination of your data in its original format and the location at which it is stored is your **Source Data System**.
+
+Your Source Data System could be CSV files in a folder, or a PostgreSQL database hosted on AWS, or any combination of numerous formats and environments.  Regardless of what your Source Data System is or where it resides, your next step in assuring its quality is implementing GX.
+
+## Getting started with Great Expectations
+
+You have data.  You care about your data.  You've made the wise decision to use Great Expectations to catch potential quality concerns with your data.  So the next thing you need to know is: _What next?_
+
+Getting started with GX can be broken down into four steps.  First, you will install GX and create a GX project.  Next, you will connect that project to your Source Data System.  Third, you will define your Expectations for the connected data. Finally, you will use GX to Validate those Expectations.
+
+Installation and Setup -> Connect to data -> Create Expectations -> Validate data
+
+### Install Great Expectations
+
+GX is a library for <PythonVersion />, and can be installed using `pip`, a library installer bundled with Python installations.  Depending on your Source Data System, you may need to install GX with some additional dependencies (generally for SQL Source Data Systems, or environments like AWS).  We streamline this process by providing special `pip` commands for specific combinations of additional dependencies.
```

Comment:
```suggestion
GX is a library for <PythonVersion />, and you can use pip to install it. If you're using a Source Data System such as SQL or AWS,  special  pip commands are provided for the installation of additional dependencies.
```
What are _additional dependencies_? Can this be defined or clarified?

Note the convention for _pip_ on the pip website:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/502093df-6d8a-4f21-a320-83e8e9df54f2)